server/src/main/generated-java/timely/api/flatbuffer/Metric.java

```diff
@@ -0,0 +1,49 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
Shouldn't these files be generated at build time, and included in the final jar rather than checked in to source control?